operations/app/terraform/modules/data_factory/pipelines.tf

```diff
@@ -0,0 +1,339 @@
+resource "azurerm_data_factory_pipeline" "transfer_files" {
+  name                = "transfer-files"
+  resource_group_name = var.resource_group_name
+  data_factory_id     = azurerm_data_factory.pdi.id
+  annotations         = []
+  concurrency         = 1
+  parameters          = {}
+
+  depends_on = [
+    azurerm_data_factory_dataset_binary.vdh,
+    azurerm_data_factory_linked_service_azure_blob_storage.pdi_datasa
+  ]
+
+  variables = {
+    "sink_base_path"   = "VIIS"
+    "source_base_path" = "/"
+  }
+
+  activities_json = jsonencode(
```

Comment:
I'm curious: do you get this JSON through an export from the data factory?

Reply:
yes. a `terraform refresh` updates the state to what is remote and a `terraform state show module.data_factory.etc.etc` shows the latest.